pquery-run.sh

```diff
@@ -460,74 +501,46 @@ if [[ $PXC -eq 1 ]]; then
   echo "wsrep-debug=1" >> ${BASEDIR}/my.cnf
   echo "innodb_file_per_table" >> ${BASEDIR}/my.cnf
   echo "innodb_autoinc_lock_mode=2" >> ${BASEDIR}/my.cnf
-  if ! check_for_version $MYSQL_VERSION "8.0.0"; then
-    # TODO: research why is this here?
-    # MariaDB: [ERROR] /test/MD300420-mariadb-10.5.3-linux-x86_64-opt/bin/mysqld: unknown variable 'innodb_locks_unsafe_for_binlog=1'
-    # echo "innodb_locks_unsafe_for_binlog=1" >> ${BASEDIR}/my.cnf
-    # echo "wsrep_sst_auth=$SUSER:$SPASS" >> ${BASEDIR}/my.cnf
-    echo "TODO"
-  else
-    echo "log-error-verbosity=1" >> ${BASEDIR}/my.cnf
-  fi
   echo "wsrep-provider=${BASEDIR}/lib/libgalera_smm.so" >> ${BASEDIR}/my.cnf
   echo "wsrep_sst_method=rsync" >> ${BASEDIR}/my.cnf
-  #echo "wsrep_sst_method=xtrabackup-v2" >> ${BASEDIR}/my.cnf
   echo "core-file" >> ${BASEDIR}/my.cnf
   echo "log-output=none" >> ${BASEDIR}/my.cnf
   echo "wsrep_slave_threads=2" >> ${BASEDIR}/my.cnf
+  echo "wsrep_on=1" >> ${BASEDIR}/my.cnf
   if [[ "$ENCRYPTION_RUN" == 1 ]]; then
     echo "log_bin=binlog" >> ${BASEDIR}/my.cnf
     echo "binlog_format=ROW" >> ${BASEDIR}/my.cnf
     echo "gtid_mode=ON" >> ${BASEDIR}/my.cnf
     echo "enforce_gtid_consistency=ON" >> ${BASEDIR}/my.cnf
     echo "master_verify_checksum=on" >> ${BASEDIR}/my.cnf
     echo "binlog_checksum=CRC32" >> ${BASEDIR}/my.cnf
-    echo "binlog_encryption=ON" >> ${BASEDIR}/my.cnf
-    echo "innodb_sys_tablespace_encrypt=ON" >> ${BASEDIR}/my.cnf
-    echo "pxc_encrypt_cluster_traffic=ON" >> ${BASEDIR}/my.cnf
+    echo "encrypt_binlog=ON" >> ${BASEDIR}/my.cnf
     if [[ $WITH_KEYRING_VAULT -ne 1 ]]; then
       echo "early-plugin-load=keyring_file.so" >> ${BASEDIR}/my.cnf
       echo "keyring_file_data=keyring" >> ${BASEDIR}/my.cnf
     fi
-  else
-    if check_for_version $MYSQL_VERSION "8.0.0"; then
-      echo "pxc_encrypt_cluster_traffic=OFF" >> ${BASEDIR}/my.cnf
-    fi
   fi
 fi
-pxc_startup() {
+
+mdg_startup() {
   IS_STARTUP=$1
   ADDR="127.0.0.1"
   RPORT=$(((RANDOM % 21 + 10) * 1000))
   SOCKET1=${RUNDIR}/${TRIAL}/node1/node1_socket.sock
   SOCKET2=${RUNDIR}/${TRIAL}/node2/node2_socket.sock
   SOCKET3=${RUNDIR}/${TRIAL}/node3/node3_socket.sock
-  if check_for_version $MYSQL_VERSION "5.7.0"; then
-    if [[ "$ENCRYPTION_RUN" == 1 ]]; then
```

Comment:
Let's perhaps leave this single line, remarked, to remind us of the possibility to use this var. Also ref comment above, perhaps we can immediately reuse this variable as an option to either use, or not use pxc_encrypt_cluster_traffic=ON

Reply:
will keep ENCRYPTION_RUN check in the code. I think MDG has different set of variables to enable for data-at-rest encryption and encryption traffic. I will look into that and update the code accordingly.